Source/WebCore/Modules/identity/DigitalCredential.cpp

```diff
@@ -98,20 +132,36 @@ void DigitalCredential::discoverFromExternalSource(const Document& document, Cre
         return;
     }
 
-    if (!document.protectedWindow()->consumeTransientActivation()) {
+    if (!window->consumeTransientActivation()) {
         promise.reject(Exception { ExceptionCode::NotAllowedError, "Calling get() needs to be triggered by an activation triggering user event."_s });
         return;
     }
 
-#if HAVE(DIGITAL_CREDENTIALS_UI)
     DigitalCredentialsRequestData requestData;
-    requestData.options = options.digital.value();
-    requestData.topOrigin = document.protectedTopOrigin()->data().isolatedCopy();
-    requestData.documentOrigin = document.protectedSecurityOrigin()->data().isolatedCopy();
-
-    Ref coordinator = document.protectedFrame()->protectedPage()->credentialRequestCoordinator();
-    if (!coordinator->presentPicker(WTFMove(promise), WTFMove(requestData), options.signal))
-        LOG(DigitalCredentials, "Failed to present the credential picker.");
+    for (auto& request : options.digital->requests) {
+        auto resultOrException = jsToCredentialRequest(document, request);
+        if (resultOrException.hasException()) {
+            promise.reject(resultOrException.releaseException());
+            return;
+        }
+
+        DigitalCredentialRequestTypes credentialVariant = resultOrException.releaseReturnValue();
+        std::visit([&](auto& credential) {
+            requestData.requests.append(credential);
+        },
+            credentialVariant);
```

Comment:
Extra leading space
```suggestion
        credentialVariant);
```

Reply:
clang-tidy keeps moving it. I'll move the lambda to the next line so it looks better.